assets/src/scripts/monitoring-location/components/hydrograph/method-picker.js

```diff
@@ -23,6 +24,7 @@ const updateAvailableMethods = function(selectElem, methods, store) {
     if (!selectedMethodID || selectedMethodID && !availableMethodIDs.includes(selectedMethodID)) {
         selectedMethodID = getPreferredIVMethodID(store.getState());
         store.dispatch(setSelectedIVMethodID(selectedMethodID));
+        showDataIndicators(false, store);
```

Comment:
Choosing a new method doesn't fetch new data but it does affect the data that is rendered, so showDataIndicators is called with isLoading false so that the No data available message gets drawn appropriately.